src/main/java/db/newLoader/SCsvLoader.java

```diff
@@ -22,8 +23,8 @@
  *
  */
 public class SCsvLoader {
-	
-	private final int threadCount = 6;//+8;
+
+	private final static int threadCount = Database.getInstance().config.loaderNbThread;
```

Comment:
Green is good.
More than red.
Red is red, so be it.

Reply:
cool !